nomenclature/cli.py

```diff
@@ -173,3 +175,63 @@ def cli_export_definitions_to_excel(
         Path and file name for the exported file
     """
     DataStructureDefinition(path / "definitions").to_excel(target)
+
+
+@cli.command("run-workflow")
+@click.argument("input_file", type=click.Path(exists=True, path_type=Path))
+@click.option(
+    "--workflow-file",
+    default=lambda: Path.cwd() / "workflow.py",
+    type=click.Path(exists=True, path_type=Path),
+)
+@click.option("--workflow-function", default="main")
+@click.option("--output-file", type=click.Path())
+def cli_run_workflow(
+    input_file: Path,
+    workflow_file: Path,
+    workflow_function: str,
+    output_file: Path | None,
+):
+    """Run a given input file through a workflow function defined in a workflow.py
+
+    Parameters
+    ----------
+    input_file : Path
+        Input data file, must be IAMC format, .xlsx or .csv
+    workflow_file : Path
+            Path to the workflow file,
+            default: current working directory / "workflow.py"
+    workflow_function : str
+        Name of the workflow function inside the workflow file, default: main
+    output_file : Path | None
+        Path to the output file where the processing results is saved, nothing
+        is saved if None is given, default: None
+
+    Raises
+    ------
+    ValueError
+        If the workflow_file does not have the specified workflow_function
+
+    Example
+    -------
+
+    Say a user is reporting issues with a Scenario Explorer processing.
+    Download the file, navigate into the workflow folder and run:
+
+    $ nomenclature run-workflow input_data.xlsx --output-file output.xlsx
+    """
+
```

Comment:
I'd prefer to avoid direct references to the "Scenario Explorer" in the nomenclature docstrings... Rather put that into the docs-user-guide, including instructions on how to install nomenclature and clone the workflow repo
```suggestion
```